src/base/bittorrent/session.h

```diff
@@ -721,6 +723,9 @@ namespace BitTorrent
 
         int m_numResumeData;
         int m_extraLimit;
+#if !(LIBTORRENT_VERSION_NUM < 10100)
+        int m_UTPClassID;
```

Comment:
if you make this the type ``libtorrent::peer_class_t`` you'll be a little bit more future proof